### DISCUSSION TITLE
Error Drawing ROMol structure in some chemical structures

### DISCUSSION THREAD

**Discussion by andresilvapimentel** [rdkit/rdkit#5157] [General]
How can I solve this error? Any suggestion?

#Drawing ROMol structure
PandasTools.AddMoleculeColumnToFrame(dfoleos, smilesCol="smiles")

#Drawing ROMol structure
PandasTools.AddMoleculeColumnToFrame(dfoleos, smilesCol="smiles")
RDKit ERROR: [19:21:28] Explicit valence for atom # 0 N, 5, is greater than permitted
RDKit ERROR: [19:21:28] Can't kekulize mol. Unkekulized atoms: 9
RDKit ERROR:
RDKit ERROR: [19:21:28] Explicit valence for atom # 10 N, 4, is greater than permitted
RDKit ERROR: [19:21:28] Explicit valence for atom # 10 N, 4, is greater than permitted
RDKit ERROR: [19:21:28] Can't kekulize mol. Unkekulized atoms: 4
RDKit ERROR:
RDKit ERROR: [19:21:28] Can't kekulize mol. Unkekulized atoms: 4
RDKit ERROR:

The chemical structures with this issue are:

[NH4][Pt]([NH4])(Cl)Cl

c1ccc(cc1)n2c(=O)c(c(=O)n2c3ccccc3)CCS(=O)c4ccccc4

Cc1cc2c(cc1C)N3C=N2[Co+]456(N7=C8[C@H](C(C7=CC9=N4C(=C(C1=N5[C@@]([C@@H]2N6C(=C8C)[C@@]([C@H]2CC(=O)N)(CCC(=O)NC[C@H](OP(=O)(O[C@@H]2[C@H](O[C@H]3[C@@H]2O)CO)[O-])C)C)([C@@]([C@@H]1CCC(=O)N)(C)CC(=O)N)C)C)[C@@]([C@@H]9CCC(=O)N)(C)CC(=O)N)(C)C)CCC(=O)N)O

Cc1cc2c(cc1C)N3C=N2[Co]456(N7=C8[C@H](C(C7=CC9=N4C(=C(C1=N5[C@@]([C@@H]2N6C(=C8C)[C@@]([C@H]2CC(=O)N)(CCC(=O)NC[C@H](OP(=O)(O[C@@H]2[C@H](O[C@H]3[C@@H]2O)CO)O)C)C)([C@@]([C@@H]1CCC(=O)N)(C)CC(=O)N)C)C)[C@@]([C@@H]9CCC(=O)N)(C)CC(=O)N)(C)C)CCC(=O)N)C#N

CCCCc1c(=O)n(n(c1=O)c2ccc(cc2)O)c3ccccc3

CCCCc1c(=O)n(n(c1=O)c2ccccc2)c3ccccc3

**Comment by wopozka** [comment]
The easiest solution? Correct structures, as non of them is correct.
